Project2015To2017/Definition/Project.cs

```diff
@@ -55,16 +55,31 @@ public DirectoryInfo NugetPackagesPath
 				var nuGetSettings = Settings.LoadDefaultSettings(projectFolder);
 				var repositoryPathSetting = SettingsUtility.GetRepositoryPath(nuGetSettings);
 
-				//return the explicitly set path, or if there isn't one, then assume the solution is one level
-				//above the project and therefore so is the 'packages' folder
-				var path = repositoryPathSetting ?? Path.GetFullPath(Path.Combine(projectFolder, @"..\packages"));
+				//return the explicitly set path, or if there isn't one, then use the solution's path if one was provided.
+				//Otherwise assume a solution is one level above the project and therefore so is the 'packages' folder
+				if (repositoryPathSetting != null)
```

Comment:
My guess is that you should actually consider solution level nuget.config first because I expect that a nuget.config in the project path or it's parents wouldn't count over the solution path if that is somewhere different.

Reply:
I think this change is correct, the closest path wins:
https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied

Reply:
Thanks, you're right, relative to the project not solution